code/modules/reagents/chemistry/recipes.dm

```diff
@@ -143,11 +143,7 @@
  * * reagent - the target reagent to convert
  */
 /datum/chemical_reaction/proc/convert_into_failed(datum/reagent/reagent, datum/reagents/holder)
```

Comment:
Why does this proc still exist?

Reply:
I have no idea, should it be removed?

Reply:
If it doesn't do anything and if we don't have failed chems then yes

Reply:
Removed!